proto/config/devconfig.proto

```diff
@@ -169,6 +169,14 @@ message EdgeDevConfig {
 
   // cluster configuration
   EdgeNodeCluster cluster = 43;
+  DhcpVendorExtensionsOverride dhcp_extensions_override = 44;
+}
+
+// DhcpVendorExtensionsOverride is used to override or add dhcp vendor extensions like f.e.
+// the NTP servers - currently no other options are supported, but DNS servers might be a
+// good candidate in the future
+message DhcpVendorExtensionsOverride {
+  bool ntpServerExclusively = 1; // use exclusively specified NTP servers
```

Comment:
the NTP servers come from here: https://github.com/lf-edge/eve-api/pull/70/files#diff-b6af1b4801076bb7a7c0b5f919ecab887a34b218f6a5f2bfdffa40845a5e618dR769-R770

Currently they are only used if there is a static network configuration - I want to change it, to even use it if the network configuration is done via dhcp.

Reply:
That link points to info.proto, i.e. status reported by device to zedcloud (we need to report these additional NTP servers, so that is correct).
But we will need such field also in this override structure or in `ipspec` which is used by `NetworkConfig`.

Reply:
Thx. I added it here: https://github.com/lf-edge/eve-api/pull/70/files#diff-35ea4ed74707f6ad6d17be169425a7acf4e25e09e9de81a0f7300ed80c5ec656R86